pyfar/classes/__init__.py

```diff
@@ -5,6 +5,7 @@
 from . import filter
 from . import transmission_matrix
 from . import warnings
+from . import _pyfar_multichannel
```

Comment:
```suggestion
```

abstract classes are not part of the official pyfar api. you can still acess them via ``pyfar.classes._pyfar_multichannel._PyfarMultichannel``